crates/config/src/lib.rs

```diff
@@ -1088,43 +1083,6 @@ impl Config {
     /// If `solc` is [`SolcReq::Local`] then this will ensure that the path exists.
     #[allow(clippy::disallowed_macros)]
```

Comment:
rm

Reply:
removed in https://github.com/foundry-rs/foundry/pull/10081/commits/165f64e6e7ca563d40964a016bade30f46ede790